src/test/java/org/pesmypetcare/webservice/dao/KcalAverageDaoTest.java

```diff
@@ -0,0 +1,376 @@
+package org.pesmypetcare.webservice.dao;
+
+import com.google.api.core.ApiFuture;
+import com.google.cloud.firestore.*;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Using the '.*' form of import should be avoided - com.google.cloud.firestore.*.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5400494)